genesis/bootstrappers.json

```diff
@@ -0,0 +1,186 @@
+            "ip": "18.162.161.230:9651"
+        },
+        {
+            "id": "NodeID-HCw7S2TVbFPDWNBo1GnFWqJ47f9rDJtt1",
+            "ip": "52.47.181.114:9651"
+        },
+        {
+            "id": "NodeID-FYv1Lb29SqMpywYXH7yNkcFAzRF2jvm3K",
+            "ip": "15.188.9.42:9651"
+        }
+    ],
+    "fuji": [
+        {
+            "id": "NodeID-2m38qc95mhHXtrhjyGbe7r2NhniqHHJRB",
+            "ip": "18.192.93.241:9651"
+        },
+        {
+            "id": "NodeID-JjvzhxnLHLUQ5HjVRkvG827ivbLXPwA9u",
+            "ip": "3.76.143.200:9651"
+        },
+        {
+            "id": "NodeID-LegbVf6qaMKcsXPnLStkdc1JVktmmiDxy",
+            "ip": "16.163.84.252:9651"
+        },
+        {
+            "id": "NodeID-HGZ8ae74J3odT8ESreAdCtdnvWG1J4X5n",
+            "ip": "13.237.111.196:9651"
+        },
+        {
+            "id": "NodeID-CYKruAjwH1BmV3m37sXNuprbr7dGQuJwG",
+            "ip": "18.167.242.179:9651"
+        },
+        {
+            "id": "NodeID-4KXitMCoE9p2BHA6VzXtaTxLoEjNDo2Pt",
+            "ip": "54.207.25.7:9651"
+        },
+        {
+            "id": "NodeID-LQwRLm4cbJ7T2kxcxp4uXCU5XD8DFrE1C",
+            "ip": "15.184.214.136:9651"
+        },
+        {
+            "id": "NodeID-4CWTbdvgXHY1CLXqQNAp22nJDo5nAmts6",
+            "ip": "13.55.124.229:9651"
+        },
+        {
+            "id": "NodeID-4QBwET5o8kUhvt9xArhir4d3R25CtmZho",
+            "ip": "3.99.55.15:9651"
+        },
+        {
+            "id": "NodeID-JyE4P8f4cTryNV8DCz2M81bMtGhFFHexG",
+            "ip": "157.241.59.198:9651"
+        },
+        {
+            "id": "NodeID-EDESh4DfZFC15i613pMtWniQ9arbBZRnL",
+            "ip": "52.29.72.46:9651"
+        },
+        {
+            "id": "NodeID-BFa1padLXBj7VHa2JYvYGzcTBPQGjPhUy",
+            "ip": "16.163.75.62:9651"
+        },
+        {
+            "id": "NodeID-CZmZ9xpCzkWqjAyS7L4htzh5Lg6kf1k18",
+            "ip": "34.248.69.195:9651"
+        },
+        {
+            "id": "NodeID-FesGqwKq7z5nPFHa5iwZctHE5EZV9Lpdq",
+            "ip": "54.66.120.144:9651"
+        },
+        {
+            "id": "NodeID-84KbQHSDnojroCVY7vQ7u9Tx7pUonPaS",
+            "ip": "3.97.132.61:9651"
+        },
+        {
+            "id": "NodeID-CTtkcXvVdhpNp6f97LEUXPwsRD3A2ZHqP",
+            "ip": "18.230.111.83:9651"
+        },
+        {
+            "id": "NodeID-hArafGhY2HFTbwaaVh1CSCUCUCiJ2Vfb",
+            "ip": "52.67.156.131:9651"
+        },
+        {
+            "id": "NodeID-4B4rc5vdD1758JSBYL1xyvE5NHGzz6xzH",
+            "ip": "176.34.80.199:9651"
+        },
+        {
+            "id": "NodeID-EzGaipqomyK9UKx9DBHV6Ky3y68hoknrF",
+            "ip": "176.34.97.64:9651"
+        },
+        {
+            "id": "NodeID-NpagUxt6KQiwPch9Sd4osv8kD1TZnkjdk",
+            "ip": "3.97.255.92:9651"
+        },
+        {
+            "id": "NodeID-3VWnZNViBP2b56QBY7pNJSLzN2rkTyqnK",
+            "ip": "15.184.142.50:9651"
+        }
+    ]
+}
```

Comment:
nit: missing new line at the end of this file (can break some random utilities that rely on this like `diff`)